spec/javascripts/enterprise_admin_spec.js

```diff
@@ -0,0 +1,71 @@
+var enrollmentApiRoot = 'http://localhost/'
+var gettext = function(arg){
+    return arg
+}
+
+describe("Manage learners admin page", function(){
+    
+    beforeEach(function(){
+        jasmine.getFixtures().fixturesPath = '__spec__/fixtures';
+        loadFixtures('learner_admin.html');
+        jasmine.Ajax.install();
+    });
+
+    afterEach(function(){
+        jasmine.Ajax.uninstall();
+    });
+
+    describe("on page load", function(){
+        it("indicates that a valid course ID is required on page load.", function(){
+            loadPage();
```

Comment:
instead of calling `loadPage();` in each function explicitly, you can put one place -- `beforeEach`